src/library_webgpu.js

```diff
@@ -2276,11 +2263,10 @@ var LibraryWebGPU = {
   },
   wgpuRenderBundleEncoderSetIndexBuffer: function(bundleId, bufferId, format, {{{ defineI64Param('offset') }}}, {{{ defineI64Param('size') }}}) {
     {{{ receiveI64ParamAsI53('offset') }}}
-    {{{ receiveI64ParamAsI32s('size') }}}
+    {{{ receiveI64ParamAsI52('size') }}}
     var pass = WebGPU.mgrRenderBundleEncoder.get(bundleId);
     var buffer = WebGPU.mgrBuffer.get(bufferId);
-    var size = {{{ gpu.convertI32PairToI53WithSentinelAsUndefined('size_low', 'size_high') }}};
-    pass["setIndexBuffer"](buffer, WebGPU.IndexFormat[format], offset, size);
+    pass["setIndexBuffer"](buffer, WebGPU.IndexFormat[format], offset, size == -1 ? undefined : size);
```

Comment:
Maybe there could be a helper function for this pattern of `x == -1 ? undefined : x`?

Reply:
Done